iModelCore/ECPresentation/Source/Content/ContentItemBuilder.cpp

```diff
@@ -19,14 +19,18 @@ USING_NAMESPACE_BENTLEY_ECPRESENTATION
 /*---------------------------------------------------------------------------------**//**
 // @bsimethod
 +---------------+---------------+---------------+---------------+---------------+------*/
-rapidjson::Document ContentValuesFormatter::GetFallbackPrimitiveValue(ECPropertyCR prop, PrimitiveType type, Utf8StringCR extendedType, IECSqlValue const& value, rapidjson::MemoryPoolAllocator<>* allocator)
+rapidjson::Document ContentValuesFormatter::GetFallbackPrimitiveValue(PrimitiveType type, Utf8StringCR extendedType, IECSqlValue const& value, rapidjson::MemoryPoolAllocator<>* allocator)
     {
     NULL_FORMATTED_PRIMITIVE_VALUE_PRECONDITION(value, type);
     rapidjson::Document json(allocator);
     ECValue v = ValueHelpers::GetECValueFromSqlValue(type, extendedType, value);
     Utf8String stringValue;
     if (v.IsNull())
         json.SetString("");
+    else if (type == PRIMITIVETYPE_DateTime)
+        {
+        json.SetString(v.GetDateTime().ToString().c_str(), json.GetAllocator()); ;
+        }

```

Comment:
In our C++ code we're not adding curly braces for one-liners. Also removed the extra semi-colon.

```suggestion
    else if (type == PRIMITIVETYPE_DateTime)
        json.SetString(v.GetDateTime().ToString().c_str(), json.GetAllocator());
```